src/interface.c

```diff
@@ -163,6 +201,24 @@ emacs_value em_cdr(emacs_env *env, emacs_value cell)
     return em_funcall(env, _cdr, 1, cell);
 }
 
+emacs_value em_list(emacs_env *env, ptrdiff_t nargs, ...)
+{
+    emacs_value args[nargs];
+
+    va_list vargs;
+    va_start(vargs, nargs);
+    for (ptrdiff_t i = 0; i < nargs; i++)
+        args[i] = va_arg(vargs, emacs_value);
+    va_end(vargs);
+
+    return env->funcall(env, _list, nargs, args);
+}
```

Comment:
There's a function for this on master now that doesn't have a varargs signature. Maybe you can rename this, or that one.